### PR TITLE
Uzbek language is being added

### PR DESCRIPTION
**Pull request by yakhyo**
I am going to add Uzbek language.

### CODE REVIEW

File: uz/Boshlang'ich Matematika/Fibonachi sonlari.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+## Fibonachi algoritmning dasturlash tillaridagi ko'rinishi:
+
+- [C++](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/math/fibonacci.cpp)
+- [Java](https://github.com/TheAlgorithms/Java/blob/master/Maths/FibonacciNumber.java)
+- [Javascript](https://github.com/TheAlgorithms/Javascript/blob/80c2dc85d714f73783f133964d6acd9b5625ddd9/Maths/Fibonacci.js)
+- [Python](https://github.com/TheAlgorithms/Python/blob/master/maths/fibonacci.py)
+
+## Video bog'ich
+
+- [Youtube](https://youtu.be/FQiRf4jb3PU)
```
Comment (Panquesito7) [inline uz/Boshlang'ich Matematika/Fibonachi sonlari.md:50]:
```suggestion
- [YouTube](https://youtu.be/FQiRf4jb3PU)
```

**Review by Panquesito7 — COMMENTED** [review]
Great work! 🚀

File: uz/Boshlang'ich Matematika/Fibonachi sonlari.md (hunk 1 above)
Comment (yakhyo) [reply uz/Boshlang'ich Matematika/Fibonachi sonlari.md:50]: Thank you.

**Review by Panquesito7 — COMMENTED** [review]
Do you have any pending changes or files to add?

**Comment by yakhyo** [comment]
So far I don't have but in near future will be several I hope.

**Comment by Panquesito7** [comment]
> So far I don't have but in near future will be several I hope.

Do you want to make another PR for the new files and merge this one or wait until you add the new ones?

**Comment by yakhyo** [comment]
I will make another PR when I made some changes.

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

**Review by vbrazo — APPROVED** [review]
LGTM 👍

**Pull request merged**